unfoldingWord/translationCore
# Update global colors and buttons

#### This pull request addresses:

Updates all colors in the core to global color variables.  Updates all buttons in the core to HTML buttons using global class names for styling.  Temporarily creates a warning for buttons called in VerseCheck.  Another PR will fix this.



#### How to test this pull request:

Open and run app and make sure everything still works and looks right!


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1659)
